.github/workflows/long_fuzz_tests_btree.yml

```diff
@@ -27,4 +27,23 @@ jobs:
     - name: Run ignored long tests
       run: cargo test -- --ignored fuzz_long
       env:
-        RUST_BACKTRACE: 1  
\ No newline at end of file
+        RUST_BACKTRACE: 1
+
+  simple-stress-test:
```

Comment:
Just confirming that your intention was to run these only when `btree.rs` changes?

Reply:
Yes kinda. Actually I was thinking it I would want to run it with vdbe changes because the other pr I opened had a related bug there.